app/grandchallenge/components/static/components/js/autocomplete_htmx.mjs

```diff
@@ -1,23 +1,20 @@
-$(document).ready(() => {
-  $(".dal-forward-conf script").text("");
-
-  $(document).on('change', 'select', (e) => {
-    const val = yl.getValueFromField(e.currentTarget);
-    $(e.currentTarget).find('option').attr('selected', false);
-    $(e.currentTarget).find(`option[value='${val}']`).attr("selected", true);
-    htmx.trigger(e.currentTarget, 'interfaceSelected');
-  });
-
-  htmx.onLoad((elem) => {
-    let vals = [];
-    $("select:disabled[name='interface'] option:selected").each((i, option) => {
-      vals.push($(option).val());
-    });
-    if (vals.length) {
-      vals = vals.map(val => `{"type": "const", "dst": "interface-${val}", "val": "${val}"}`);
-    }
-    vals.push(`{"type": "const", "dst": "reader-study", "val": "${$('#readerStudySlug').data('slug')}"}`);
-    $(".dal-forward-conf script").text(`[${vals.join(',')}]`);
-  });
-
-});
+$(document).ready(() => {

```

Comment:
In general it would be good to do things without JQuery (although we cannot remove it as we're using a large tables app built with it). Most things should be available as native browser functionality now, see https://youmightnotneedjquery.com.